treeherder/webapp/api/classifiedfailure.py

```diff
@@ -1,6 +1,6 @@
 from collections import defaultdict
 
-import rest_framework_filters as filters
+import django_filters
```

Comment:
For this and the ones below, I'd perhaps use `from django_filters import rest_framework as filters`, like so:
https://django-filter.readthedocs.io/en/develop/guide/rest_framework.html#quickstart